### PR TITLE
SDP3x Airspeed Correction

### PR DESCRIPTION
**Pull request by acfloria**
- Update the model for the airspeed correction provided by [sensirion](https://github.com/PX4/Firmware/issues/8140).

- Add a pressure drop compensation which can be used for other pitot configurations than the standard drotek one.

- Remove the ets_airspeed driver from px4fmu-v2_default

I did test flights with a membrane based sensor, our custom pitot tube and the drotek pitot tube. During the flights I mainly did loitering with different airspeeds. If the airspeed has an offset then the wind will have an oscillation with the loitering frequency.

Wind estimate for the membrane based sensor:
membrane

Wind estimate for the drotek pitot tube:
drotek

Wind estimate for our custom pitot tube:
custom

Based on the wind plots I would say the airspeed is still by about 0.2 to 0.4 m/s off but this can be compensated by fine tuning the parameters.

### CODE REVIEW

File: src/modules/systemlib/airspeed.c [hunk 1]
```diff
@@ -55,59 +55,107 @@
...
-			double flow_SDP33 = (300.805 - 300.878 / (0.00344205 * pow(dp, 0.68698) + 1)) * 1.29 / rho_air;
+			// assumes a metal pitot tube with round tip as here: https://drotek.com/shop/2986-large_default/sdp3x-airspeed-sensor-kit-sdp31.jpg
+			// and tubing as provided by px4/drotek (1.5 mm diameter)
+			// The tube_len represents the length of the tubes connecting the pitot to the sensor.
+			switch (pmodel) {
+			case AIRSPEED_COMPENSATION_MODEL_PITOT:
+			case AIRSPEED_COMPENSATION_MODEL_NO_PITOT: {
+					const double dp_corr = (double)dp * 96600.0 / (double)pressure_ambient;
+					// flow through sensor
+					double flow_SDP33 = (300.805 - 300.878 / (0.00344205 * pow(dp_corr, 0.68698) + 1)) * 1.29 / rho_air;
```
Comment (dagar) [inline src/modules/systemlib/airspeed.c]: Do we actually need double precision?

Comment (acfloria) [reply src/modules/systemlib/airspeed.c]: I am not sure, but I will check it.

Comment (acfloria) [reply src/modules/systemlib/airspeed.c]: I changed it and did a test flight with it. I saw no issue so it should be fine.

**Comment by dagar** [comment]
The timing isn't ideal, but we should consider getting this in for v1.7.0 (the first stable release that supports the sdp3x). Otherwise bringing this in later will break existing setups, not to mention I already see people independently discovering problems when swapping out the pitot tube.

**Comment by LorenzMeier** [comment]
Let's get it in!

**Comment by supergra** [comment]
What units should `CAL_AIR_TUBELEN` be given in?

The parameter reference states "meters", and none of the code indicates anything otherwise. I don't see a conversion.

However, the python code above is clearly saying that it's in mm, and the equations look identical to what's in the code.

**Pull request merged**